src/objects/zcl_abapgit_object_doma.clas.abap

```diff
@@ -82,10 +83,13 @@ CLASS ZCL_ABAPGIT_OBJECT_DOMA IMPLEMENTATION.
       " Domain values
       lt_dd07v_tmp = it_dd07v.
       LOOP AT lt_dd07v_tmp ASSIGNING <ls_dd07v>.
+        lv_valpos = <ls_dd07v>-valpos.
         READ TABLE lt_dd07_texts ASSIGNING <ls_dd07_text>
-          WITH KEY ddlanguage = <lv_lang> valpos = <ls_dd07v>-valpos.
+          WITH KEY ddlanguage = <lv_lang> domvalue_l = <ls_dd07v>-domvalue_l
+                   domvalue_h = <ls_dd07v>-domvalue_h.
```

Comment:
is this change related to clearing of the fields?

Reply:
Yes, because "it_dd07_texts" don't have valpos filled. I'm not sure if we can use "sy-tabix" to read, probably it'll work but I think reading using the key is most reliable

Reply:
hmm